taxi/views.py

```diff
@@ -34,6 +35,26 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
```

Comment:
Using `fields = "__all__"` in `CreateView` and `UpdateView` exposes all fields of the `Manufacturer` model in the form. Ensure that this is intentional and that no sensitive fields are included. Consider specifying only the fields you want to expose.